config/config.yaml

```diff
@@ -72,8 +75,8 @@ utils:
     use_HTTP_chromadb_client: True   # recommended: True (use http client for the chromadb vectorstore?)
     # use_HTTP_chromadb_client: False
     vectordb_update_time: 10
-    chromadb_host: "localhost"
-    chromadb_port: 8000
+    chromadb_host: chromadb
```

Comment:
How does this work? I thought that the chromadb host was the IP address of where the chromadb server was running. In the past, I used localhost as an alias to 127.0.0.1. 

It could be correct (no need to necessarily change it), I just wanna make sure I understand it.

Reply:
Great question! This is a Docker (Compose)-specific setting, and when we support multiple deployment scenarios, we'll need to have separate configs to handle this.

In Docker Compose, when a set of containers/services are defined in a `compose.yaml` file, each service can access ports on each other service by using the name(s) assigned in the `compose.yaml`. In our case, I named the Chroma service `chromadb` in our `compose.yaml`, so the application code running inside e.g. the mailbox, data manager, etc. accesses ChromaDB on `chromadb:PORT`.